Packs/Gamma/Integrations/Gamma/Gamma.yml

```diff
@@ -0,0 +1,107 @@
+category: DLP
```

Comment:
This is not an existent category, please select from the available categories.

Reply:
DLP is missing from existing categories list, we were told to submit this new category as part of the PR by the Partnership Success team. Please let me know if this is an issue, and I can change it to something else